source/shared/net/desktop-wallet/account-recovery.rst

```diff
@@ -0,0 +1,41 @@
+.. _account-recovery-desktop:
+
+=====================
+Account recovery
+=====================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Account recovery in the desktop wallet
+=======================================================
+
+The recovery process is used to recover lost accounts using only the ledger device, which was used to create those accounts.
+
+It should only be necessary to perform this recovery if you have lost the backup of your accounts and identities. To learn more about making backups of your data, see :ref:`Make a backup of identities, accounts, and addresses <export-import-desktop>`.
+
+If the user has lost their ledger device or the device has ceased to function, the keys can be restored in another ledger device, by initialising it with the same recovery phrase as the original device. To learn more about the recovery phrase of a ledger device:
+`What is a recovery phrase <https://www.ledger.com/academy/crypto/what-is-a-recovery-phrase/>`_ .
+
+   .. Note::
+      This recovery process will recover your accounts, but it is not possible to recover the identities used to create these accounts, because they do not exists on the chain, to learn more about identities see :ref:`Identities and accounts <reference-id-accounts>`.
+      The names of the accounts, and the notes on the credentials are also not recoverable, as these are only saved locally.
+
+The recovery process
+--------------------------------------
+The keys, which are used to sign account transactions, are attached to credentials, and those credentials each belong to an identity.
+
+In the desktop wallet, when we create a new identity, it has an associated number on the ledger, which we refer to as the identity index. These indices are used sequentially, so the first identity created from a ledger uses index *0*, the next uses index *1* and so forth.
+
+The recovery process takes advantage of that and also checks the indices sequentially. Unfornately the identity process can fail, which could cause an index to be skipped, so the you must determine yourself when the recovery is finished. If you have never had your identity issuance fail, you can stop the recovery when you encounter an unused index.
```

Comment:
```suggestion
The recovery process uses the indices and also checks them sequentially. Unfortunately, the identity issuance process can fail and cause an index to be skipped, and therefore, you must determine yourself when the recovery is finished. If you've never had your identity issuance fail, you can stop the recovery when you encounter an unused index.
```

Reply:
It's a bit unclear what "The recovery process takes advantage of that" actually means, so I tried to rephrase it a bit.

Reply:
Because we have the indices, we can check the identities sequentially, maybe like this:
```suggestion
The recovery process checks the identities sequentially, by using the indices. Unfortunately, the identity issuance process can fail and cause an index to be skipped, and therefore, you must determine yourself when the recovery is finished. If you've never had your identity issuance fail, you can stop the recovery when you encounter an unused index.
```

Reply:
This is fine. I removed a comma.